vmdk_plugin/plugin.go

```diff
@@ -249,10 +249,9 @@ func (d *vmdkDriver) Create(r volume.Request) volume.Response {
 	if err != nil {
 		log.WithFields(log.Fields{"name": r.Name,
 			"error": err}).Error("Attach volume failed, removing the volume ")
-		err = d.ops.Remove(r.Name, nil)
-		if err != nil {
-			log.WithFields(log.Fields{"name": r.Name, "error": err}).Error("Remove volume failed ")
-			return volume.Response{Err: err.Error()}
+		errRm := d.ops.Remove(r.Name, nil)
+		if errRm != nil {
+			log.WithFields(log.Fields{"name": r.Name, "error": errRm}).Error("Remove volume failed ")
```

Comment:
Yes. I think errors on rollback are generally warnings - like it's helpful to warn if something we expected to complete failed, but it does not impact success/failure of the actual operation requested.  We can silently ignore but I am generally not comfortable with silent failures.